qlog/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "qlog"
-version = "0.15.1"
+version = "0.15.2"
```

Comment:
Would you mind releasing `qlog` `v0.15.2` with this change? That would allow me to downstream `qlog` `v0.15` into Firefox.

Reply:
Yes, but I think this will confuse cargo release, so it should be removed.